scripts/automation/Whitesource_container_scan.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Please use `Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates`

Reply:
changed it